### ISSUE TITLE
Question about training loss

### ISSUE THREAD

**Issue by zjr2000** [JetAstra/SDAR#27]
Hi,

I’m reproducing continued pre-training with LLaMA Factory, initializing from qwen3-base. The loss starts around 30 in the first steps, which is much higher than with standard AR training. When I initialize from your released checkpoints, the loss starts around 8, which also seems relatively high.

From your experience, is this expected behavior, or does it suggest a configuration issue on my side?

Thanks!

**Comment by zjr2000** [comment]
Hi,

I found that the loss scales with `gradient_accumulation_steps`, similar to https://github.com/huggingface/transformers/issues/40564 and https://github.com/hiyouga/LLaMA-Factory/issues/7743. When I set `accepts_loss_kwargs` to `False`, the behavior goes back to normal. Here is the code I changed:

```python
class SDARModel(SDARPreTrainedModel):
    accepts_loss_kwargs = False
    ...

class SDARForCausalLM(SDARPreTrainedModel, GenerationMixin):
    _tied_weights_keys = ["lm_head.weight"]
    _tp_plan = {"lm_head": "colwise_rep"}
    _pp_plan = {"lm_head": (["hidden_states"], ["logits"])}
    accepts_loss_kwargs = False
    ...
```

I don’t think `loss_kwargs` are actually used in the model, so maybe you could merge this change into your code.
